lib/response.js

```diff
@@ -25,7 +25,7 @@ var sign = require('cookie-signature').sign;
 var normalizeType = require('./utils').normalizeType;
 var normalizeTypes = require('./utils').normalizeTypes;
 var setCharset = require('./utils').setCharset;
-var statusCodes = http.STATUS_CODES;
+var statusCodes = require('statuses/codes')
```

Comment:
Lol, yeah totally missed the mechanism that exposed that on the first read.  Sorry for the bother.  Updating now.